switchsai/sailag.c

```diff
@@ -149,6 +149,16 @@ static sai_status_t sai_create_lag(_Out_ sai_object_id_t* lag_id,
 
   lag_h = attribute->value.oid;
 
+  attribute =
+      get_attr_from_list(SAI_LAG_ATTR_PORT_VLAN_ID, attr_list, attr_count);
+  if (attribute == NULL) {
+    status = SAI_STATUS_INVALID_PARAMETER;
```

Comment:
Only attributes that are _MANDATORY_ON_CREATE_ should return _SAI_STATUS_MANDATORY_ATTRIBUTE_MISSING_ in case of failure. Since, these are not mandatory attributes, I don't think that's a valid return type.